include/ram_pwrdn.rst

```diff
@@ -0,0 +1,22 @@
+.. _lib_ram_pwrdn:
+
+RAM Power Down
+##############
+
+The RAM Power Down library is a basic module for disabling unused sections of RAM, which allows to save power in low power applications.
+
+To disable unused RAM sections, call :cpp:func:`power_down_unused_ram`.
+This function will automatically disable all memory regions that are not used by the application.
+
+.. note:: If the RAM memory is used indirectly (e.g. through the pointer) it could happen that this region will be powered off by :cpp:func:`power_down_unused_ram`. Next access to that memory region would result in the Bus fault exception.
```

Comment:
After discussing on Teams:
```suggestion
.. note::
    Make sure you are not writing to memory indirectly, for example through pointers.
    :cpp:func:`power_down_unused_ram` will also power down regions that are using RAM memory indirectly, which can result in a bus fault exception on the next access.
```

Reply:
Done.

Reply:
@greg-fer This is not precise. You can access memory through pointer, e.g when the pointer points to some variable known to linker. If you access memory outside the range linker knows is being used, you may access ram region which is powered off and bus fault will occur.